sql/mozfun/norm/browser_version_info/udf.sql

```diff
@@ -3,14 +3,18 @@ RETURNS STRUCT<
   version STRING,
   major_version NUMERIC,
   minor_version NUMERIC,
+  patch_version NUMERIC,
   is_major_release BOOLEAN
 > AS (
   STRUCT(
     version_string AS version,
-    norm.truncate_version(version_string, 'major') AS major_version,
-    norm.truncate_version(version_string, 'minor') AS minor_version,
-    ARRAY_LENGTH(SPLIT(version_string, '.')) = 2
-    AND ENDS_WITH(version_string, '.0') AS is_major_release
+    norm.extract_version(version_string, 'major') AS major_version,
+    norm.extract_version(version_string, 'minor') AS minor_version,
+    norm.extract_version(version_string, 'patch') AS patch_version,
```

Comment:
I checked where this is used. Looks like this is only used in views so adding `patch_version` should not cause schemas to break